src/ui/pages/IncomingRequest/components/BallotTransactionRequest.scss

```diff
@@ -0,0 +1,88 @@
+.ballot-transaction {
+  & .page-header {
+    ion-toolbar {
+      --background: transparent;
+
+      ion-title.md {
+        width: calc(100% - 3.75rem);
+      }
+    }
+  }
+
+  .ballot-transaction-header {
+    margin: 1.75rem 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .ballot-owner-logo {
+      width: 5rem;
+      height: 5rem;
+      border-radius: 2.5rem;
+      margin-bottom: 0.75rem;
+    }
+
+    .ballot-name {
+      font-size: 1.5rem;
+      line-height: 1.75rem;
+      margin-bottom: 0.5rem;
+      color: var(--ion-color-primary);
+    }
+
+    .ballot-subtitle {
+      font-weight: 500;
+      font-size: 1rem;
+      line-height: 1.25rem;
+      margin-bottom: 0.5rem;
+      color: var(--ion-color-secondary);
+    }
+
+    .ballot-link {
+      font-weight: 500;
+      font-size: 0.75rem;
+      line-height: 1rem;
+      margin: 0;
+      color: rgba(54, 60, 74, 0.75);
+    }
+  }
+
+  .ballot-transaction-content {
+    margin-bottom: 3rem;
+
+    .ballot-address {
+      margin-bottom: 1.5rem;
+
+      .card-details-info-block-inner {
+        padding: 1.25rem 1.5rem;
+
+        @media screen and (min-width: 250px) and (max-width: 370px) {
+          font-size: 0.8rem;
+          padding: 0.85rem 1.25rem;
+        }
+      }
+    }
```

Comment:
`CardDetailsBlock` and `CardDetailsItem` are shared component. Same as with the `PageHeader` before, we should make sure that the component works fine everywhere. Making adjustments in the page that uses it should only be an exception.